sp800-63a/sec5_proofing.md

```diff
@@ -87,42 +97,44 @@ The goal of identity validation is to confirm and establish a linkage between th
-	- The CSP SHOULD allow two (2) attempts for an applicant to complete the KBV.  A CSP MAY allow no more than three (3) attempts to complete the KBV.
-	- The CSP MAY use KBV to verify an applicant's identity against only one (1) piece of validated identity evidence.  
-	- The CSP SHALL NOT present diversionary KBV questions.  The CSP SHALL NOT allow answers to KBV questions be 'None of the Above', 'Not Applicable (N/A)', or similar to be regarded as correct.
+- KBV SHALL NOT be based on data held by the issuing source of an applicant's supplied identity evidence. **MG: I think I get the point here, but does it mean that if the dmv is the issuing source, and the dmv knows everything about my car ownership and driving history, then nothing ever involving a car or driving can be used?**
+- The CSP SHALL only use information that is expected to be known only to the applicant and the source, to include any information needed to trigger the KBV process. Information accessible freely or for any fee in the public domain SHALL NOT be used. **MG: Not sure I get the "trigger" part. Isn't it just resolution that triggers (I think I'd prefer "seed") the process?**
+- The CSP SHALL allow a resolved, validated, or verified identity to opt-out of KBV and leverage another process for verification.
+- The CSP SHOULD perform KBV by verifying knowledge of recent transactional history that the CSP is a participant in.  The CSP SHALL ensure that transaction information has at least 20 bits of entropy. For example, to reach minimum entropy requirements, the CSP could ask the applicant for verification of the amount(s) of a micro-deposit(s) to a valid bank account, so long as the total number of digits is seven or greater. **MG: is this better?** 
+- The CSP MAY perform KBV by asking the applicant questions to demonstrate they are the owner of the claimed information. However, the following requirements apply:
+	- The CSP SHALL require a minimum of four KBV questions with each requiring a correct answer to successfully complete the KBV step.
+	- The CSP SHOULD require a free form response to a KBV question.  The CSP MAY allow multiple choice answers, however, if multiple choice answer are provided, the CSP SHALL require a minimum of four answer options per question.
+	- The CSP SHOULD allow two attempts for an applicant to complete the KBV.  A CSP SHALL NOT allow more than three attempts to complete the KBV.
+	- The CSP MAY use KBV to verify an applicant's identity against only one piece of validated identity evidence.  
+	- The CSP SHALL NOT present diversionary KBV questions.  The CSP SHALL NOT allow answers to KBV questions be 'None of the Above', 'Not Applicable (N/A)', or similar to be regarded as correct. **MG: Brent was telling me diversionary questions actually statistically less likely for an attacker to get right. Do we have any data either way?**
```

Comment:
I thought in a pilot we heard otherwise, but maybe it was when they were all diversionary.  Then the bad guys were just clicking NA for everything and getting in.